data/common.yaml

```diff
@@ -1,8 +1,2 @@
 ---
-lldpd::manage_facts: true
-lldpd::manage_service: true
 lldpd::manage_jq: true
-lldpd::manage_repo: true
```

Comment:
Should this be removed or kept?

Reply:
otherwise we would need to add more yaml files and enable it explicitly for most distros. I think it should stay there.

Reply:
Right, but currently it's being removed.

Reply:
ugh :D right. I fixed it